code/crates/test/app/src/app.rs

```diff
@@ -216,8 +218,13 @@ pub async fn run(state: &mut State, channels: &mut Channels<TestContext>) -> eyr
                             .get_validator_set(state.current_height)
                             .expect("Validator set should be available");
 
+                        let updates = Updates {
+                            validator_set: Some(validator_set),
```

Comment:
Could we change the code here to compare old and new validator sets and pass None if the same? This way we can test the `None` case for valset in malachite.

Reply:
https://github.com/circlefin/malachite/pull/1227/commits/1f14c3f7a4af732bd4a6949272d10a01bc4a82a9